src/components/application_manager/src/help_prompt_manager_impl.cc

```diff
@@ -31,6 +31,10 @@
 */
 
 #include "application_manager/help_prompt_manager_impl.h"
+
+#include <algorithm>
+#include <iterator>
+
```

Comment:
extra space

Reply:
@mrapitis it's not an extra space as we are using blank lines to separate `include` groups according to our coding style. Please check for more details here:
https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes